test/response.spec.js

```diff
@@ -234,7 +234,7 @@ describe('Testing Response', () => {
 
   it('Testing pruneResponse false', (done) => {
     api.wrap('GET test', [
-      api.FieldsParam('fields', 'query', { fields: ['foo'], autoPrune: false })
+      api.FieldsParam('fields', 'query', { fields: ['foo'], autoPrune: null })
```

Comment:
And then change the name of the test
```suggestion
      api.FieldsParam('fields', 'query', { fields: ['foo'] })
```